### PR TITLE
created the homepage

### PR DESCRIPTION
**Pull request by rtaha07**

### CODE REVIEW

**Review by garber34 — COMMENTED** [review]
Can we go over this is a group? I think we should discuss whether this is going to be a single page application.  And I don't really understand what styled-components, react-script, and react-scroll are doing

**Comment by rtaha07** [comment]
I finished the main home page and added the sign-in and sign-up pages. Please review it and tell me if anything needs to be changed.

File: client/components/HeroSection/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  HeroBg,
+  HeroContainer,
+  VideoBg,
+  HeroContent,
+  HeroH1,
+  HeroP,
+  HeroBtnWrapper,
+} from './HeroElements';
+import Video from '../../videos/video.mp4';
+import { Button } from '../ButtonElements';
```
Comment (garber34) [inline client/components/HeroSection/index.js]: ButtonElements not found. It looks like it got renamed to ButtonElement

File: client/components/ProductsSamples/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+// import image6 from '../../images/flowers6.jpg';
+import image2 from '../../images/flowers2.jpg';
```
Comment (garber34) [inline client/components/ProductsSamples/index.js]: Getting this error: 

Module parse failed: Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type.

File: client/components/ProductsSamples/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// import image3 from '../../images/flowers3.jpg';
+import image4 from '../../images/flowers4.jpg';
```
Comment (garber34) [inline client/components/ProductsSamples/index.js]: Getting error:
Module parse failed: Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type.

File: client/components/HeroSection/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import {
+  HeroBg,
+  HeroContainer,
+  VideoBg,
+  HeroContent,
+  HeroH1,
+  HeroP,
+  HeroBtnWrapper,
+} from './HeroElements';
+import Video from '../../videos/video.mp4';
```
Comment (garber34) [inline client/components/HeroSection/index.js]: Getting error:

Module parse failed: Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type.

**Review by garber34 — DISMISSED** [review]
I'm errors when trying to merge with dev because  there are multiple modules named NavBar and Categories

File: client/components/HeroSection/index.js (hunk 1 above)
Comment (rtaha07) [reply client/components/HeroSection/index.js]: Oh. that's right, I will correct it. Thanks Jeff.

File: client/components/ProductsSamples/index.js (hunk 3 above)
Comment (rtaha07) [reply client/components/ProductsSamples/index.js]: I don't know why. I did not get this error in my local machine. I will check it and see what I can do. Thanks

**Comment by rtaha07** [comment]
I will delete the extra modules, if that is ok with Lawrence @OneExceptionn

File: client/components/Categories/index.js [hunk 5]
```diff
@@ -1,11 +1,47 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { HashRouter as Router, Link, Route } from 'react-router-dom';
+import { getCategories } from '../store/store'; //may change if store is broken out
```
Comment (garber34) [inline client/components/Categories/index.js]: this module is broken. i think it should be ../../store/store.js

**Review by garber34 — COMMENTED** [review]
To get rid of the compile errors I had to move the image and video assets into the public folder, so that the client would have access to them. Then updated the file paths

**Comment by rtaha07** [comment]
Hey guys. Please, somebody has to review Design branch and approve it, so that I can merge it into dev. Thanks

**Review by garber34 — APPROVED** [review]
I fixed the merge conflicts and simplified the folder structure to make try to prevent more conflicts in the future.

I commented out the "Categories" tab in the navbar, that should be a dropdown menu to filter the products page, not a link to another page.

**Pull request merged**